### PR TITLE
Create documentation issue template

### PR DESCRIPTION
**Pull request by Spice-King**
As per the [conversation here](https://discord.com/channels/170995199584108546/893621333538258944/1003434850692386856), here is the PR for adding a documentation issue template, with a bit of back and forth done with mxzf to take off the rougher edges.

### CODE REVIEW

**Comment by aaclayton** [comment]
Assigning this to @cswendrowski to look at and discuss with the team.

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Documentation errors
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:1]: Should likely also encapsulate requests for new documentation

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+description: Let us know about typos and goofs in our site, documentation or API documentation blocks
+title: "[Docs]: "
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:3]: We ended up not using this convention, you can remove this line

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+labels: ["documentation", "api"]
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:4]:
```suggestion
labels: ["documentation"]
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    attributes:
+      value: |
+        Use this form to report a mistake or discrepancy with the core Foundry software site,
+        documentation, or code documentation comments.
+  - type: dropdown
+    id: course-where
+    attributes:
+      label: Where did you find it?
+      options: 
+       - Main Foundry site (not package listings)
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:16]:
```suggestion
       - Knowledge Base (https://foundryvtt.com/kb/)
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+       - API site
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:17]:
```suggestion
       - V10 API Documentation (https://foundryvtt.com/api/v10/)
       - V9 API Documentation (https://foundryvtt.com/api/)
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+       - Code documentation comments
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:18]:
```suggestion
       - Code documentation comments (foundry.js)
```

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        Any issues with the content on pages for packages should be addressed to the authors
```
Comment (cswendrowski) [inline .github/ISSUE_TEMPLATE/documentation.yml:24]: Great callout

**Review by cswendrowski — COMMENTED** [review]
Nice work, have some minor suggestions. Will review with the team later today

File: .github/ISSUE_TEMPLATE/documentation.yml (hunk 4 above)
Comment (Spice-King) [reply .github/ISSUE_TEMPLATE/documentation.yml:16]: Slight disagree, though calling out the KB by name is one thing, there is the rest of the site, where there is a non-zero chance for typos. Meet in the middle?
```diff
-       - Main Foundry site (not package listings)
+       - Knowledge Base (https://foundryvtt.com/kb/)
+       - Other Foundry site page (not package listings)
 ```

File: .github/ISSUE_TEMPLATE/documentation.yml (hunk 7 above)
Comment (Spice-King) [reply .github/ISSUE_TEMPLATE/documentation.yml:24]: Your goddamn right

**Pull request merged**